doc/release-notes.md

```diff
@@ -61,7 +61,8 @@ RPC changes
 
 ### Low-level changes
 
-- The `fundrawtransaction` rpc will reject the previously deprecated `reserveChangeKey` option.
+- The `fundrawtransaction` RPC will reject the previously deprecated `reserveChangeKey` option.
+- The `createrawtransaction` RPC will now accept an array or dictionary (kept for compatibility) for the `outputs` parameter. This makes the result of the call deterministic.
```

Comment:
Could use simpler language and be more explicit `makes the result of the call deterministic` -> `means the order of transaction outputs can be specified by the client` or similar